angrmanagement/ui/widgets/qdecomp_options.py

```diff
@@ -41,11 +45,29 @@ def __init__(self, parent, option, type_: int, enabled=True):
         else:
             raise NotImplementedError("Unsupported option type %s." % self.type_)
 
-        self.setFlags(self.flags() | Qt.ItemIsUserCheckable)
-        if enabled:
-            self.setCheckState(0, Qt.Checked)
+        # should make a dropdown option
+        if hasattr(self.option, "value_type") and self.option.value_type != bool and self.option.candidate_values:
+            self._combo_box = QComboBox()
+            self._combo_box.addItems(self.option.candidate_values)
+            self._combo_box.setToolTip("%s: %s" % (option.NAME, option.DESCRIPTION))
+            # XXX: causes an itemChanged event for the tree
+            self._combo_box.currentTextChanged.connect(lambda x: self.setText(0, self._combo_box.currentText()))
```

Comment:
This line is hacky and I am open to change if anyone has any. Essentially we need to trigger a currentItemChanged event from the combobox change events.